OpenSim/Common/Component.cpp

```diff
@@ -202,19 +208,25 @@ void Component::connect(Component &root)
 
 
 // Call connect on all components and find unconnected Connectors a
```

Comment:
Comment is incomplete. Also, I don't see where unconnected Connectors are detected, either here or in `connect()`.